### PR TITLE
Adding network policy creation

### PR DESCRIPTION
**Pull request by rashm1n**
$Subect

### CODE REVIEW

**Review by BLasan — APPROVED** [review]
LGTM! @ashera96 please review

File: impl/createApi.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 var dirPath string
 var desFilePath string
 
-func CreateAPI(filePath, namespace, serviceUrl, apiName, version string, isDryRun bool) {
+func CreateAPI(filePath, namespace, serviceUrl, apiName, version string, isDryRun bool, applyNetworkPolicy bool) {
```
Comment (ashera96) [inline impl/createApi.go]:
```suggestion
func CreateAPI(filePath, namespace, serviceUrl, apiName, version string, isDryRun, applyNetworkPolicy bool) {
```

File: impl/createApi.go [hunk 2]
```diff
@@ -264,13 +264,13 @@ func createAndDeploySwaggerAPI(swaggerSpec spec.Swagger, filePath, namespace, se
...
-		handleDeploy(file, filePath, namespace, swaggerSpec.Info.Title, swaggerSpec.Info.Version, swaggerSpec, utils.Swagger2)
+		handleDeploy(file, filePath, namespace, swaggerSpec.Info.Title, swaggerSpec.Info.Version, swaggerSpec, utils.Swagger2, parsedURL, applyNetworkPolicy)
 	} else {
-		handleDryRun(file, filePath, namespace, swaggerSpec.Info.Title, swaggerSpec.Info.Version, swaggerSpec, utils.Swagger2)
+		handleDryRun(file, filePath, namespace, swaggerSpec.Info.Title, swaggerSpec.Info.Version, swaggerSpec, utils.Swagger2, parsedURL, applyNetworkPolicy)
 	}
 }
 
-func createAndDeployOpenAPI(openAPISpec openapi3.T, filePath, namespace, serviceUrl, apiName string, isDryRun bool) {
+func createAndDeployOpenAPI(openAPISpec openapi3.T, filePath, namespace, serviceUrl, apiName string, isDryRun bool, applyNetworkPolicy bool) {
```
Comment (ashera96) [inline impl/createApi.go:251]:
```suggestion
func createAndDeployOpenAPI(openAPISpec openapi3.T, filePath, namespace, serviceUrl, apiName string, isDryRun, applyNetworkPolicy bool) {
```

**Comment by BLasan** [comment]
@rashm1n Please resolve the merge conflicts

**Comment by rashm1n** [comment]
Closing due to conflicts, will be tracked from
* https://github.com/BLasan/APKCTL-Demo/pull/32

**Pull request closed without merging**